composer.json

```diff
@@ -12,9 +12,7 @@
     }
   ],
   "require": {
-    "php": ">=5.6"
-  },
-  "require": {
+    "php": ">=7.0",
     "phpunit/phpunit": "5.*"
```

Comment:
@WillJW The PHPUnit bit belongs in `require-dev`. The PHP part should be fine where it is.